api/catalog/api/views/health_views.py

```diff
@@ -37,5 +47,6 @@ def _check_es(self) -> Response | None:
     def get(self, request: Request):
         if "check_es" in request.query_params:
             self._check_es()
+        self._check_db()
```

Comment:
We should create an exception class (like `ElasticsearchHealthcheckException`) for the database as well, so that we can return a 503 response when the DB connection fails.

Also shouldn't this also be behind a flag? I guess maybe not, because Django wouldn't be up unless the DB connection succeeds.

Reply:
The native function already [handles DB](https://github.com/django/django/blob/5da5f3773ec281a027f99c9b56add47d076650b5/django/db/backends/base/base.py#L285) [specific errors](https://github.com/django/django/blob/5da5f3773ec281a027f99c9b56add47d076650b5/django/db/backends/base/base.py#L677) for us.